Dockerfile

```
@@ -75,6 +75,9 @@ WORKDIR ${WORK_DIR}
 COPY --chown=user:user . ${SRC_DIR}
 
 # installs buildozer and dependencies
-RUN pip3 install --user Cython==0.28.6 ${SRC_DIR}
+RUN pip3 install --user wheel Cython==0.28.6 ${SRC_DIR}
+
+# Upgrade virtualenv and pip
+RUN pip3 install -U pip virtualenv
```

Thanks for the update. How about in-lining the entire thing?
```suggestion
RUN pip3 install --user --upgrade Cython==0.28.6 wheel pip virtualenv ${SRC_DIR}
```

---

Yeah look really better, I just build the image and compile my app, it's ok for me !